### PR TITLE
enable ssl for development 

### PR DESCRIPTION
**Pull request by alfredodeza**
With a self-signed cert.

### CODE REVIEW

File: deploy/playbooks/roles/common/tasks/nginx.yml [hunk 1]
```diff
@@ -33,6 +33,27 @@
...
+  action: file dest=/etc/ssl/certs state=directory
+  sudo: true
+
+- name: ensure ssl private directory
+  action: file dest=/etc/ssl/private state=directory
+  sudo: true
+
+- name: create self-signed SSL cert
+  command: openssl req -new -nodes -x509 -subj "/C=US/ST=Oregon/L=Portland/O=IT/CN=${app_name}" -days 3650 -keyout /etc/ssl/private/{{ ansible_fqdn }}.key -out /etc/ssl/certs/{{ ansible_fqdn }}-bundled.crt -extensions v3_ca creates=/etc/nginx/ssl/{{ ansible_fqdn }}-bundled.crt
+  when: development_server
```
Comment (ktdreyer) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: Is this going to regenerate a new SSL cert and key file every time Ansible runs?

Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: yes  -_-

Comment (ktdreyer) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: Can we make it conditional on like "if the files don't exist" or something?

Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: yep working on it, sorry, my -_- meant 'how could I miss this'

**Comment by ktdreyer** [comment]
Looks good! thank you

**Pull request merged**